src/app/components/hris/employees/employee-profile/accordions/accordion-career-summary/accordion-certificates/accordion-certificates.component.ts

```diff
@@ -228,7 +237,10 @@ export class AccordionCertificatesComponent {
       next: () => {
         this.snackBarService.showSnackbar("Certificate deleted", "snack-success");
         this.copyOfCertificates.splice(index, 1);
-        this.employeeCertificates.splice(index, 1);
+        this.sharedAccordionFunctionality.employeeCertificates.splice(index, 1);
+        this.sharedAccordionFunctionality.employeeCertificatesFields = this.sharedAccordionFunctionality.employeeCertificatesFields - 4;
+        this.sharedAccordionFunctionality.calculateCareerCertficatesFormProgress();
+        this.sharedAccordionFunctionality.totalCareerProgress();
```

Comment:
Refactor:
What happens when when index is out of range? say if index =99999 or -3